contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -381,14 +386,9 @@ var BaseWorkspaceView = BaseView.extend({
 	},
 	open_archive:function(){
 		var ArchiveView = require("edit_channel/archive/views");
-		window.current_channel.get_root("trash_tree").fetch({
-			success:function(fetched){
-				window.current_channel.set("trash_tree", fetched.attributes);
-				var archive = new ArchiveView.ArchiveModalView({
-					model : fetched
-			 	});
-			}
-		});
+		var archive = new ArchiveView.ArchiveModalView({
```

Comment:
Don't wait to fetch the trash tree, it should be updated automatically. This is to address the problem where a user can double click the Trash button and end up opening two modals